### PR TITLE
Fifo animal shelter

### PR DESCRIPTION
**Pull request by mattburger**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/test/java/code401Challenges/fifoAnimalShelter/TestAnimalShelter.java [hunk 1]
```diff
@@ -1,4 +1,33 @@
 package code401Challenges.fifoAnimalShelter;
 
+import code401Challenges.stackandqueues.Queue;
```
Comment (evanslaton) [inline code401Challenges/src/test/java/code401Challenges/fifoAnimalShelter/TestAnimalShelter.java]: Unused import!
